celery_tasks.py

```diff
@@ -56,9 +55,10 @@ def create_pull_request_for_issue(*, token, owner, repo_name, issue, comments=No
             logger.info(f"Ignoring issue #{issue['number']} as @aiderbot was not mentioned")
             return {"message": "Issue ignored as @aiderbot was not mentioned"}, 200
 
-        if ALLOWED_USERNAME and issue['user']['login'] != ALLOWED_USERNAME:
-            logger.info(f"Ignoring issue from non-allowed user: {issue['user']['login']}")
-            return {"message": "Issue from non-allowed user ignored"}, 200
+        user_permission = github_api.get_user_permission(token, owner, repo_name, issue['user']['login'])
```

Comment:
@Aiderbot it looks like 'author_association' is a part of the webhook payload, e.g. if the user is the owner of the repo, then issue['author_association'] will be 'OWNER'. We can use that instead of doing another API call.